README.md

```diff
@@ -111,3 +111,29 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+Run:
+```Shell
+tools/setup_env.sh
```

Comment:
I'm fine with that.

Getting off topic a bit: we should decide what happens on a developer's machine and what happens in docker, in any case.

* Installing git and git-secrets feels like a dev host machine concern
* Installing angular CLI feels "in the grey area" for me
* Installing gradle and maven feels like a "docker dev env" thing to me